setup.py

```diff
@@ -69,58 +69,93 @@
 
 Have fun!
 """
 
 from __future__ import print_function
 
-from ooni import __version__, __author__
 import os
-import sys
-import shutil
+import errno
 import tempfile
-import subprocess
+from glob import glob
+
 from ConfigParser import SafeConfigParser
 
 from os.path import join as pj
-from setuptools import setup, Command
-from setuptools.command.install import install
-from distutils.spawn import find_executable
+from setuptools import setup
+from setuptools.command.install import install as InstallCommand
+from subprocess import check_call
 
-GEOIP_ASN_URL = "https://download.maxmind.com/download/geoip/database/asnum/GeoIPASNum.dat.gz"
-GEOIP_URL = "https://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz"
-TEST_LISTS_URL = "https://github.com/citizenlab/test-lists/archive/master.zip"
+from ooni import __version__, __author__
+
+CLASSIFIERS = """\
+Development Status :: 5 - Production/Stable
+Environment :: Console
+Framework :: Twisted
+Intended Audience :: Developers
+Intended Audience :: Education
+Intended Audience :: End Users/Desktop
+Intended Audience :: Information Technology
+Intended Audience :: Science/Research
+Intended Audience :: Telecommunications Industry
+License :: OSI Approved :: BSD Licens
+Programming Language :: Python
+Programming Language :: Python :: 2
+Programming Language :: Python :: 2 :: Only
+Programming Language :: Python :: 2.6
+Programming Language :: Python :: 2.7
+Operating System :: MacOS :: MacOS X
+Operating System :: POSIX
+Operating System :: POSIX :: BSD
+Operating System :: POSIX :: BSD :: BSD/OS
+Operating System :: POSIX :: BSD :: FreeBSD
+Operating System :: POSIX :: BSD :: NetBSD
+Operating System :: POSIX :: BSD :: OpenBSD
+Operating System :: POSIX :: Linux
+Operating System :: Unix
+Topic :: Scientific/Engineering :: Information Analysis
+Topic :: Security
+Topic :: Security :: Cryptography
+Topic :: Software Development :: Libraries :: Application Frameworks
+Topic :: Software Development :: Libraries :: Python Modules
+Topic :: Software Development :: Testing
+Topic :: Software Development :: Testing :: Traffic Generation
+Topic :: System :: Networking :: Monitoring
+"""
 
-def run_command(args, cwd=None):
-    try:
-        p = subprocess.Popen(args, stdout=subprocess.PIPE, cwd=cwd)
-    except EnvironmentError:
-        return None
-    stdout = p.communicate()[0].strip()
-    if p.returncode != 0:
-        return None
-    return stdout
 
 def is_lepidopter():
-    if os.path.exists("/etc/default/lepidopter"):
-        return True
-    return False
+    return os.path.exists('/etc/default/lepidopter')
+
+
+def is_updater_installed():
+    return os.path.exists('/etc/lepidopter-update/version')
 
-class OoniInstall(install):
+
+def install_lepidopter_update():
+    check_call(["data/lepidopter-update.py", "install"])
+
+
+def mkdir_p(path):
+    try:
+        os.makedirs(path)
+    except OSError as ose:
+        if ose.errno != errno.EEXIST:
+            raise
```

Comment:
It's better to not use inside of the setup.py stuff from your package as the package may have depedencies that are not met at install time.